### DISCUSSION TITLE
How to add bio next to my profile pic?

### DISCUSSION THREAD

**Discussion by YoItsMEALohelo** [community/community#40518] [New to GitHub] [answered]
how can I add bio that stays next to my profile picture?

**Comment by MitkoVtori** [comment, marked as answer]
create a repository named with your username, then the README.md file will show up next to your profile picture and you only have to edit it.

**Reply by DSDmark** [reply]
Hi @YoItsMEALohelo,

also read this for more details:-https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme
